masonry/src/widget/sized_box.rs

```diff
@@ -520,6 +587,24 @@ impl Widget for SizedBox {
 
     fn paint(&mut self, ctx: &mut PaintCtx, scene: &mut Scene) {
         let corner_radius = self.corner_radius;
+        let size = ctx.size();
+
+        // Paint shadow if present
+        if let Some(shadow) = &self.shadow {
+            let shadow_rect = size
+                .to_rect()
+                .inset(-shadow.spread_radius)
+                .to_rounded_rect(corner_radius);
+
+            scene.draw_blurred_rounded_rect_in(
+                &shadow_rect,
+                Affine::translate(shadow.offset),
+                shadow_rect.rect(),
+                shadow.color,
+                shadow.corner_radius.unwrap_or(corner_radius.top_left),
+                shadow.blur_radius,
+            );
```

Comment:
Why are you using the clipped version of the blurred rect? The `_in` version. The unclipped version is how I would expect it to look.
<img width="313" alt="image" src="https://github.com/user-attachments/assets/a78d2849-d16f-4e5d-964b-250aa3efccf9" />